NEWS-remake.md

```diff
@@ -1,7 +1,43 @@
-Version 4.3.1+dbg-1.4
+Here we have note remake changes. For changes to GNU Make see its NEWS file.
+
+Version 4.3.1+dbg-1.5
 ======================
 
-Rebase code on GNU Make 4.3. This was made possible by Thomas Kupper (boretom) who did all the heavy lifting while rocky looked on in amazement.
+Rebase code on GNU Make 4.3.
+
+This was made possible by Thomas Kupper (boretom) who did all the heavy lifting while rocky looked on in amazement.
+
+Thomas also added OS/X CI testing, and tested on the BSD-ish platforms.
+
+With this release, we've started redoing the texinfo documentation in [sphinx](https://www.sphinx-doc.org) and put on [readthedocs](https://remake.readthedocs.io/en/latest/). Since sphinx supports output into LaTeX, TeXinfo, pdf, epub, and info as well as HTML, the TeXinfo document will be removed when the sphinx documentation is more stable. I expect the in next release to work on this.
+
+Similarly, you'll see that [`README.md`](https://github.com/rocky/remake) has been redone and includes screencasts. The install docs have also been corrected and expanded.
+
+New and Changed Features
+-------------------------
+
+* [`--search parent`](https://remake.readthedocs.io/en/latest/features.html#searching-for-a-makefile-in-parent-directories) -- I really like this one.
+* [`--tasks`](https://remake.readthedocs.io/en/latest/features.html#listing-and-documenting-makefile-targets) -- the definition of a "tasks" has been simplified.
+  Now, it is simply a target that has a description comment (`#:`) before it.
+  After many years of using this myself, I highly encourage people to start using decription comments more.
+  Output from runing `remake --tasks` is nicer because we use spaces to align columns rather than tabs.
```

Comment:
"runing" > "running"

Reply:
Thanks. I'm appreciative of any/all help I get. 
you don't really have to mention my for every little thing I did
> 
Acknowledging it is a way of showing sincere appreciation.